### PR TITLE
refactor(editor): Add telemetry for SSO/SAML (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
https://linear.app/n8n/issue/PAY-1142

### CODE REVIEW

File: packages/editor-ui/src/views/SettingsSso.vue [hunk 1]
```diff
@@ -115,8 +128,10 @@ const onTest = async () => {
 		if (typeof window !== 'undefined') {
 			window.open(url, '_blank');
 		}
+		testSucceeded.value = true;
```
Comment (ivov) [inline packages/editor-ui/src/views/SettingsSso.vue:131]: Not quite sure here.

File: packages/editor-ui/src/views/SettingsSso.vue [hunk 2]
```diff
@@ -78,6 +82,8 @@ const getSamlConfig = async () => {
 	ssoSettingsSaved.value = !!config?.metadata;
 };
 
+const testSucceeded = ref(false);
```
Comment (netroy) [inline packages/editor-ui/src/views/SettingsSso.vue:85]: what's the benefit of using a reactive reference over a regular old `let` here?

Comment (ivov) [reply packages/editor-ui/src/views/SettingsSso.vue]: My bad! Must've been tired EoD

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.22.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.22.0)

**Pull request merged**